operator/api/redpanda/v1alpha3/nodepool_types.go

```diff
@@ -75,12 +75,14 @@ type EmbeddedNodePoolSpec struct {
 }
 
 type BrokerTemplate struct {
-	Image     string                      `json:"image"`
-	Resources corev1.ResourceRequirements `json:"resources"`
+	Images    BrokerImages                 `json:"images"`
+	Resources *corev1.ResourceRequirements `json:"resources"`
 	// Arguments to be passed to rpk tune
 	// https://docs.redpanda.com/current/reference/rpk/rpk-redpanda/rpk-redpanda-tune/
-	Tuning                    []string               `json:"tuning"`
-	NodeConfig                map[string]ValueSource `json:"nodeConfig"`
+	Tuning []string `json:"tuning"`
+
+	NodeConfig map[string]ValueSource `json:"nodeConfig"`
+	// TODO: Need to figure out what exactly we want to do with this
 	RPKConfig                 map[string]ValueSource `json:"rpkConfig"`
```

Comment:
If this is problematic I think we could safely drop it or otherwise merge it into the NodeConfig.

I left it in place so the webhooks would have somewhere to "stash" so round trip conversion would work as expected.